### PR TITLE
DOC: Update notes for ITKExamples -> ITKSphinxExamples repository renaming

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: Documentation/ReleaseNotes/5.0.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Unpack optional testing data in the same directory where the Library Source is u
...
... in the development of ITK: The official repository was moved to GitHub! This move includes [ITK's main repository](https://github.com/InsightSoftwareConsortium/ITK), as well as many other ITK projects such as the [ITKSphinxExamples](https://github.com/InsightSoftwareConsortium/ITKSphinxExamples), and many [ITK remote modules](https://github.com/InsightSoftwareConsortium). It is now very easy for anyone to participate in the development of the library. This has already been a success with many new authors getting involved and submitting patches through the GitHub pull-request mechanism. This change also includes moving the issue tracker to the [GitHub issue tracker](https://github.com/InsightSoftwareConsortium/ITK/issues) where any problem encountered with ITK should be reported. ITK's GitHub migration was explained in more detail in [ITK 5.0 Beta 3: GitHub](https://discourse.itk.org/t/itk-5-0-beta-3-github/1504). Finally, moving ITK to GitHub made it straightforward to use Microsoft Azure Pipeline Continuous Integration platform: Each pull-request is tested on all 3 major platforms (Windows, Linux, MacOS) and Python builds can be requested to ensure that a patch can be integrated.
```
Comment (dzenanz) [inline Documentation/ReleaseNotes/5.0.md:48]: As the old links redirect to the new repo, maybe we should not even update those links?

File: Examples/README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
-ITK Examples Directory
+ITK Sphinx Examples Directory
 ----------------------
```
Comment (dzenanz) [inline Examples/README.md:2]: Add more ---

Should we switch to `# Title` notation?

**Review by dzenanz — CHANGES_REQUESTED** [review]

File: Documentation/ReleaseNotes/4.12.md [hunk 3]
```diff
@@ -566,7 +566,7 @@ Matt McCormick (32):
 `     ENH: Add .sha512 content links`\
 `     ENH: Remove .md5 content links`\
 `     ENH: Update ExternalData_URL_TEMPLATES for sha512 resources`\
-`     ENH: Update the ITKExamples scripts to use the .sha512 content links`\
+`     ENH: Update the ITKSphinxExamples scripts to use the .sha512 content links`\
```
[the remark this answers is not in the document]
Comment (thewtex) [reply Documentation/ReleaseNotes/4.12.md:569]: Yes, I will revert all the changes to the release notes.

File: Documentation/ReleaseNotes/5.0.md (hunk 1 above)
Comment (thewtex) [reply Documentation/ReleaseNotes/5.0.md:48]: I've reverted the changes to the release notes.

File: Examples/README.md (hunk 2 above)
Comment (thewtex) [reply Examples/README.md:2]: :+1: Updated

**Review by dzenanz — APPROVED** [review]

File: Examples/README.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
-ITK Examples Directory
-----------------------
+ITK Sphinx Examples Directory
+-----------------------------
```
Comment (jhlegarreta) [inline Examples/README.md:2]: The examples in this folder do not correspond to the Sphinx examples, but are intended to provide the examples for the SW Guide so I would not change the title. @thewtex probably revert this change?

File: Utilities/KWStyle/KWStyle.cmake [hunk 5]
```diff
@@ -4,10 +4,10 @@ if(ITK_USE_KWSTYLE)
     ${ITK_SOURCE_DIR}/Utilities/KWStyle/ITK.kws.xml
     )
   set(kwstyle_itk_examples_files_list_file
-    ${ITK_BINARY_DIR}/Utilities/KWStyle/ITKExamplesFiles.txt
+    ${ITK_BINARY_DIR}/Utilities/KWStyle/ITKSphinxExamplesFiles.txt
```
Comment (jhlegarreta) [inline Utilities/KWStyle/KWStyle.cmake:7]: Same comment for the contents of that will populate this text file.

File: Utilities/KWStyle/KWStyle.cmake [hunk 6]
```diff
@@ -4,10 +4,10 @@ if(ITK_USE_KWSTYLE)
...
     )
   configure_file( # KWStyle requires that the files list be absolute paths
-    ${ITK_SOURCE_DIR}/Utilities/KWStyle/ITKExamplesFiles.txt.in
+    ${ITK_SOURCE_DIR}/Utilities/KWStyle/ITKSphinxExamplesFiles.txt.in
```
Comment (jhlegarreta) [inline Utilities/KWStyle/KWStyle.cmake:10]: Same comment here: see [ITKExamplesFiles.txt.in](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Utilities/KWStyle/ITKExamplesFiles.txt.in).

**Review by jhlegarreta — CHANGES_REQUESTED** [review]
I might be wrong, but I guess some changes were inadvertently introduced. See comments inline.

File: Examples/README.md (hunk 4 above)
Comment (thewtex) [reply Examples/README.md:2]: @jhlegarreta good catch, thanks!

Overall, this naming change will provide more differentiation and help avoid confusions with the sphinx examples / software guide examples.

File: Utilities/KWStyle/KWStyle.cmake (hunk 5 above)
Comment (thewtex) [reply Utilities/KWStyle/KWStyle.cmake:7]: :+1: good catch.

File: Utilities/KWStyle/KWStyle.cmake (hunk 6 above)
Comment (thewtex) [reply Utilities/KWStyle/KWStyle.cmake:10]: Done

**Review by jhlegarreta — APPROVED** [review]
